source/documentation/xsd-reference.html.md.erb

```diff
@@ -20,41 +20,43 @@ Use these to validate your XML. Please note, there are some known omissions. The
 
 You can ignore this field.
 
-### [Download zip file](ctc-traders-api-xsds.zip) with all the CTC Traders API XSDs.
+### [Download zip file](https://developer.service.hmrc.gov.uk/guides/common-transit-convention-traders-service-guide/documentation/ctc-traders-api-xsds.zip) with all the CTC Traders API XSDs.
 
 Alternatively you can review each XSD file individually using the links below.
 
 ## Validate XML posted into the API:
 
 | **Title** | **Description** |
 |------|-------------|
-|**[Arrival notification message CC007A (IE007)](https://github.com/hmrc/common-transit-convention-traders/blob/master/conf/xsd/cc007a.xsd)** |Send an Arrival Notification message from the trader at destination to the office at destination|
-|**[Declaration data cancellation request CC014A (IE014)](https://github.com/hmrc/common-transit-convention-traders/blob/master/conf/xsd/cc014a.xsd)**| Send a message from the trader at departure asking the office at departure to cancel declaration data|
-|**[Declaration data message CC015B (IE015)](https://github.com/hmrc/common-transit-convention-traders/blob/master/conf/xsd/cc015b.xsd)**|Send a declaration data message from the trader at departure to the office at departure so they can can start the transit|
-|**[Unloading remarks message CC044A (IE044)](https://github.com/hmrc/common-transit-convention-traders/blob/master/conf/xsd/cc044a.xsd)** |Send a message from the trader at destination to the office at to let the office at destination know that the goods have been unloaded (IE044)|
+|**[Arrival notification message CC007A (IE007)](https://github.com/hmrc/common-transit-convention-traders/blob/master/conf/xsd/cc007a.xsd)** |Message type sent by the trader to the office of departure containing an arrival notification|
+|**[Declaration data cancellation request CC014A (IE014)](https://github.com/hmrc/common-transit-convention-traders/blob/master/conf/xsd/cc014a.xsd)**|Message type sent by the trader to the office of departure containing a cancellation request|
+|**[Declaration data message CC015B (IE015)](https://github.com/hmrc/common-transit-convention-traders/blob/master/conf/xsd/cc015b.xsd)**|Message type sent by the trader to the office of departure to indicate that they would like to start a movement|
+|**[Unloading remarks message CC044A (IE044)](https://github.com/hmrc/common-transit-convention-traders/blob/master/conf/xsd/cc044a.xsd)** |Message type sent by the trader to the office at destination to let them know that the goods have been unloaded (IE044)|
```

Comment:
I know it has always been like this but should we really be pointing people to `master`?

Reply:
Unless they are hosted elsewhere, what options do we have?

Reply:
We could link to a recent tag, e.g. https://github.com/hmrc/common-transit-convention-traders/blob/v0.240.0/conf/xsd/cc014a.xsd